### ISSUE TITLE
Code signing certificate is expired

### ISSUE THREAD

**Issue by koenrh** [bkuhlmann/git-cop#32]
## Overview

The certificate that is used by users to verify the signature of signed gems has expired. 

## Steps to Recreate

1. Download the developer's code signing certificate: `curl https://www.alchemists.io/gem-public.pem -o git-cop.pem`
2. Attempt to install a version of the gem that is signed: `gem install git-cop -v 1.6.0 --trust-policy MediumSecurity`.
3. Note the error:
```
ERROR:  While executing gem ... (Gem::Security::Exception)
    certificate /CN=brooke/DC=alchemists/DC=io not valid after 2017-10-19 16:44:13 UTC
```
3. Alternatively, view the certificate's contents (using `openssl x509 -in git-cop.pem -noout -text`), and note the expiration date (Not After).

```
Certificate:
    Data:
        Version: 3 (0x2)
        Serial Number: 1 (0x1)
    Signature Algorithm: sha1WithRSAEncryption
        Issuer: CN=brooke, DC=alchemists, DC=io
        Validity
            Not Before: Oct 19 16:44:13 2016 GMT
            Not After : Oct 19 16:44:13 2017 GMT
        Subject: CN=brooke, DC=alchemists, DC=io

...
```

**Comment by bkuhlmann** [comment]
Thanks, yes, I will work on getting this fixed. This is actually a bigger issues as it affects all of my published gems.

**Comment by bkuhlmann** [comment]
:information_source: This is fixed in [Version 1.6.2](https://github.com/bkuhlmann/git-cop/releases/tag/v1.6.2).
